src/main/java/net/atos/zac/policy/input/DocumentInput.java

```diff
@@ -14,7 +15,7 @@ public class DocumentInput extends UserInput {
     @JsonbProperty("document")
     private final DocumentData documentData;
 
-    public DocumentInput(final LoggedInUser loggedInUser, final DocumentData documentData) {
+    public DocumentInput(@NotNull LoggedInUser loggedInUser, final DocumentData documentData) {
```

Comment:
Can still be final no?



```suggestion
    public DocumentInput(@NotNull final LoggedInUser loggedInUser, final DocumentData documentData) {
```

Reply:
Got rid of these